recipes/dirent/all/test_package/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.1)
 project(test_package)
 
 set(CMAKE_VERBOSE_MAKEFILE TRUE)
```

Comment:
### 
```suggestion
```

Reply:
Do you know that you can directly edit files, without using suggestions ? You simply have to click on the 3 dots on top right of each file.

Reply:
Thanks for the tip, @ericLemanissier ! I used that feature a while ago, but it will definitely help!